components/LinkCustom/LinkCustom.tsx

```diff
@@ -0,0 +1,36 @@
+// Import necessary modules from Next.js and React
+import Link from 'next/link';
+import React from 'react';
+
+interface LinkCustomProps {
+  href: string;
+  children: any;
```

Comment:
S: Is there a way to make this not be an `any ` type?  Like `HTML Element | string` or  `React.HTML.Element | string`?

Reply:
Yes this is a good idea; thank you.

Reply:
Done.